### PR TITLE
zh-translation: /news/releases/0.x/announcing-0.3/index.md #1521

### PR DESCRIPTION
**Pull request by gorda**
Ref: [ServiceMesher#1521](https://github.com/servicemesher/istio-official-translation/issues/1521)

### CODE REVIEW

File: content/zh/news/releases/0.x/announcing-0.3/announcing-0.3/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## 网络{#networking}
+
+- **TCP Egress 规则**。现在，您可以指定影响 TCP 级别流量的 Egress 规则。
+
+## 策略执行和遥测{#policy-enforcement-and-telemetry}
+
+- **改善缓存**。Envoy 和 Mixer 之间的缓存得到了很大改善，大大降低了授权检查的平均延迟。
+
+- **改进的列表适配器**。Mixer “列表” 适配器现在支持正则表达式匹配。关详细信息，请参见适配器的 [配置选项](/zh/docs/reference/config/policy-and-telemetry/adapters/list/)。
```
Comment (yuxiaobo96) [inline content/zh/news/releases/0.x/announcing-0.3/announcing-0.3/index.md]:
```suggestion
- **改进的列表适配器**。Mixer “列表” 适配器现在支持正则表达式匹配。有关详细信息，请参见适配器的[配置选项](/zh/docs/reference/config/policy-and-telemetry/adapters/list/)。
```

File: content/zh/news/releases/0.x/announcing-0.3/announcing-0.3/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+- **配置校验**。Mixer 对配置状态进行更广泛的验证，以便更早发现问题。我们希望在即将发布的版本中，投入更多的精力在此功能上。
+
+如果您想了解细节，可以在 [这里](https://github.com/istio/istio/wiki/v0.3.0) 查看我们更详细的低级发行说明。
```
Comment (yuxiaobo96) [inline content/zh/news/releases/0.x/announcing-0.3/announcing-0.3/index.md]:
```suggestion
如果您想了解细节，可以在[这里](https://github.com/istio/istio/wiki/v0.3.0) 查看我们更详细的低级发行说明。
```

Comment (gorda) [reply content/zh/news/releases/0.x/announcing-0.3/announcing-0.3/index.md]: @yuxiaobo96 已修改，感谢细心 review

**Comment by yuxiaobo96** [comment]
@rootsongjc 
已修改，可以合并

**Pull request merged**